plugins/nvm/nvm.plugin.zsh

```diff
@@ -1,8 +1,17 @@
 # Set NVM_DIR if it isn't already defined
 [[ -z "$NVM_DIR" ]] && export NVM_DIR="$HOME/.nvm"
 
-# Try to load nvm only if command not already available
 if ! type "nvm" &> /dev/null; then
-    # Load nvm if it exists
-    [[ -f "$NVM_DIR/nvm.sh" ]] && source "$NVM_DIR/nvm.sh"
+    # Load if it exists
+    if [ ! -f "/usr/local/opt/nvm" ]; then
+        # brew install nvm
+        nvm_sh_dir="/usr/local/opt/nvm"
+        # This loads nvm
+        [ -s "${nvm_sh_dir}/nvm.sh" ] && . "${nvm_sh_dir}/nvm.sh"
+        # This loads nvm bash_completion
+        [ -s "${nvm_sh_dir}/etc/bash_completion" ] && . "${nvm_sh_dir}/etc/bash_completion"
```

Comment:
On my machine, the bash completion lives at

${nvm_sh_dir}/etc/bash_completion.d/nvm

Reply:
You are right
I found this to be a mistake of `brew info nvm`

I installed **zsh-completions**, it will point to `/usr/local/etc/bash_completion.d/nvm`, so it is normal on my machine.